src/common/components/ui/formik/RemovableField.css

```
@@ -0,0 +1,9 @@
+.remove-button--tooltip {
```

we should try to follow BEM
```suggestion
.remove-button__tooltip {
```
```suggestion
.remove-button--tooltip {
```